spec/factories.rb

```diff
@@ -0,0 +1,33 @@
+FactoryGirl.define do
+  sequence :lat do |n|
+    "#{n}.0".to_f
+  end
+
+  sequence :lon do |n|
+    "#{n}.0".to_f
```

Comment:
Yet another way would be `Faker::Address.latitude` and `Faker::Address.longitude`